scripts/build.d/qt

```diff
@@ -11,14 +11,14 @@ if [ -z "${SYNCGIT}" ]; then
   pkgfull=$pkgname-$pkgver
   pkgfoler=qt-everywhere-src-$pkgver
 
-  pkgfn=$pkgfull.zip
-  pkgurl=https://download.qt.io/official_releases/qt/$pkgmajorver/$pkgver/single/qt-everywhere-src-$pkgver.zip
+  pkgfn=$pkgfull.tar.xz
+  pkgurl=https://download.qt.io/official_releases/qt/$pkgmajorver/$pkgver/single/qt-everywhere-src-$pkgver.tar.xz
 
   download_md5 $pkgfn $pkgurl $pkgmd5
```

Comment:
Why isn't the md5 hash updated?

Reply:
ah… I forgot it.

Reply:
And if md5 hash is wrong, `download_md5` seems not to terminate the build process right ?
it just show `md5 hash …. but got ….`

Reply:
Yes, that's by design.  Many tarballs are large and hashing can only be calculated after it is fully downloaded.  It wastes too much bandwidth to discard the downloaded file or stop the process.